ts/src/test/static/response/binance.json

```diff
@@ -910,7 +910,7 @@
                         "unrealizedPnl": 0.51106678,
                         "leverage": null,
                         "liquidationPrice": 64846.9875379,
-                        "collateral": 25.72470371,
```

Comment:
this is unrelated

Reply:
oh accidentally committed in this branch